pkg/controller/direct/bigqueryconnection/connection_mapping.go

```diff
@@ -289,3 +288,26 @@ func BigQueryConnectionConnectionSpec_ToProto(mapCtx *direct.MapContext, in *krm
 	// MISSING: SalesforceDataCloud
 	return out
 }
+
+func CloudSqlCredential_FromProto(mapCtx *direct.MapContext, in *pb.CloudSqlCredential) *krm.CloudSqlCredential {
+	if in == nil {
+		return nil
+	}
+	out := &krm.CloudSqlCredential{}
+	// This is sensitive data, input-only field.
+	// out.Username = direct.LazyPtr(in.GetUsername())
```

Comment:
Does this cause the round-trip tests to fail?

Reply:
We don't have roundtrip test for this resource yet. I think we cannot not map input-only field back to KRM any ways, can we?

Reply:
We'll have to figure out to do here.  We'll probably just have to skip the field.  That's acceptable I think.